source/standalone/workflows/rl_games/play.py

```diff
@@ -155,7 +155,7 @@ def main():
             # convert obs to agent format
             obs = agent.obs_to_torch(obs)
             # agent stepping
-            actions = agent.get_action(obs, is_deterministic=True)
+            actions = agent.get_action(obs, is_deterministic=agent.is_deterministic)
```

Comment:
For `Factory` environments, we evaluate policies stochastically. 

`agent.is_deterministic` is specified in the `rl_games_ppo_cfg.yaml` under `params.config.player.deterministic`. If this is not specified, the default value will be `True` -- as such, this change should not affect other environments.